client/src/components/Sidebar/Sidebar.tsx

```diff
@@ -58,6 +58,11 @@ export const Sidebar: React.FunctionComponent<SidebarProps> = (
   // space next to the content
   const [peeking, setPeeking] = React.useState(false);
 
+  // 'open' indicates whether the sidebar is visible or not.
+  // this is meant to be used to show/hide the menu on small screens.
+  // it only has effect on small screens.
+  const [open, setOpen] = React.useState(false);
```

Comment:
Could we rename `open` to something more intuitive here? I feel like the set of variables we've got to describe the menu state is getting slightly confusing with this addition (since I'd probably expect `open` to be eg the opposite of `collapsed`, but they're really describing different things). Maybe `visible`?

Reply:
Oh, good point! Yep I've renamed this to `visibleOnMobile`